### PR TITLE
update challenges.yml and config.yml

### PR DESCRIPTION
**Pull request by dutchy1001**
Removed the "completedChallenges" comments in the challenges.yml
Add info for island showing in GUI or not in config.yml
Translation tag added (hope this time it's right)

### CODE REVIEW

File: uSkyBlock-Core/src/main/java/us/talabrek/ultimateskyblock/command/island/InfoCommand.java [hunk 1]
```diff
@@ -76,7 +76,7 @@ public void run() {
                                     score.getScore(), score.getCount(),
                                     VaultHandler.getItemName(score.getBlock())));
                         }
-                        player.sendMessage(String.format("\u00a7aIsland level is %5.2f", getState().getScore()));
+                        player.sendMessage (tr(String.format("\u00a7aIsland level is %5.2f", getState().getScore())));
```
Comment (rlf) [inline uSkyBlock-Core/src/main/java/us/talabrek/ultimateskyblock/command/island/InfoCommand.java]: Should be:
```java
player.sendMessage(tr("\u00a7aIsland level is {0,number,###.##}", getState().getScore()));
```

Comment (dutchy1001) [inline uSkyBlock-Core/src/main/java/us/talabrek/ultimateskyblock/command/island/InfoCommand.java]: I didn't know that "String.format" had to be removed as this was there as original code or that " %5.2f" changed to {0,number,###.##}. 
Guess that is my downfall of not knowing how to code properly :)

Comment (rlf) [inline uSkyBlock-Core/src/main/java/us/talabrek/ultimateskyblock/command/island/InfoCommand.java]: Ye, `String.format` uses another formatting-mode than `java.text.MessageFormat` does - the latter is the one we use in GNU Gettext (i18n).

**Comment by dutchy1001** [comment]
still not right. fixed in next pull request

**Pull request closed without merging**